### ISSUE TITLE
MalformedURLException thrown on paths starting with `http`

### ISSUE THREAD

**Issue by propensive** [olafurpg/scalafmt#205]
This stack trace:
```
Exception in thread "main" java.net.MalformedURLException: no protocol: http/src/file.scala
        at java.net.URL.<init>(URL.java:586)
        at java.net.URL.<init>(URL.java:483)
        at java.net.URL.<init>(URL.java:432)
        at scala.io.Source$.fromURL(Source.scala:131)
        at org.scalafmt.util.FileOps$.readFile(FileOps.scala:35)
        at org.scalafmt.cli.Cli$$anonfun$getCode$1$$anonfun$apply$3.apply(Cli.scala:187)
        at org.scalafmt.cli.Cli$$anonfun$getCode$1$$anonfun$apply$3.apply(Cli.scala:186)
        at scala.collection.TraversableLike$WithFilter$$anonfun$map$2.apply(TraversableLike.scala:683)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
        at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
        at scala.collection.TraversableLike$WithFilter.map(TraversableLike.scala:682)
        at org.scalafmt.cli.Cli$$anonfun$getCode$1.apply(Cli.scala:185)
        at org.scalafmt.cli.Cli$$anonfun$getCode$1.apply(Cli.scala:184)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.mutable.ArraySeq.foreach(ArraySeq.scala:74)
        at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:241)
        at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
        at org.scalafmt.cli.Cli$.getCode(Cli.scala:184)
        at org.scalafmt.cli.Cli$.run(Cli.scala:195)
        at org.scalafmt.cli.Cli$$anonfun$main$1.apply(Cli.scala:240)
        at org.scalafmt.cli.Cli$$anonfun$main$1.apply(Cli.scala:240)
        at scala.Option.foreach(Option.scala:257)
        at org.scalafmt.cli.Cli$.main(Cli.scala:240)
        at org.scalafmt.cli.Cli.main(Cli.scala)
```
can be recreated with the following:
```
mkdir -p http/src
touch http/src/file.scala
scalafmt -f http/src
```

**Comment by olafurpg** [comment]
[That was sloppy of me](https://github.com/olafurpg/scalafmt/blob/a7ccedcdb6b3df1ce9ac61f003d9e22cb027df8a/core/src/main/scala/org/scalafmt/util/FileOps.scala#L34), will be an easy fix. Thanks!

**Comment by olafurpg** [comment]
Closed by #206
